### PR TITLE
Added leitstellenspiel.de support

### PR DESCRIPTION
**Pull request by bastianleicht**
image
image
image

### CODE REVIEW

File: websites/L/Leitstellenspiel/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "www.jogo-operador112.com",
+    "www.operador193.com",
+    "www.centro-de-mando.mx",
+    "www.dyspetcher101-game.com",
+    "www.missionchief-japan.com"
+  ],
+  "description": {
+    "de": "Bei Leitstellenspiel sitzt Du in der Leitstelle, fährst Einsätze auf einer echten Karte und meisterst alleine oder zusammen mit Freunden viele Einsätze!",
+    "en": "Find yourself in a dispatch center and manage emergencies with your friends on a real map!"
+  }
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/dist/metadata.json:41]: add the descriptions in the other languages

File: websites/L/Leitstellenspiel/presence.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  document.location.pathname === "/" ||
+  document.location.pathname === ""
+) 
+  presenceData.details = "Manages Emergencies";
+ else if (document.location.pathname.startsWith("/einsaetze")) 
+  presenceData.details = "Viewing all possible scenarios";
+ else if (document.location.pathname.startsWith("/toplist")) 
+  presenceData.details = "Viewing the Toplist";
+ else 
+  presenceData.details = "Homepage";
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/presence.ts:20]: Can you show anything else?

**Review by Bas950 — CHANGES_REQUESTED** [review]
Add altnames.

File: websites/L/Leitstellenspiel/presence.ts (hunk 2 above)
Comment (bastianleicht) [reply websites/L/Leitstellenspiel/presence.ts:20]: No

File: websites/L/Leitstellenspiel/dist/metadata.json (hunk 1 above)
Comment (bastianleicht) [reply websites/L/Leitstellenspiel/dist/metadata.json]: I added some more in the latest Commit.

File: websites/L/Leitstellenspiel/dist/metadata.json [hunk 3]
```diff
@@ -37,6 +37,18 @@
...
   "description": {
     "de": "Bei Leitstellenspiel sitzt Du in der Leitstelle, fährst Einsätze auf einer echten Karte und meisterst alleine oder zusammen mit Freunden viele Einsätze!",
-    "en": "Find yourself in a dispatch center and manage emergencies with your friends on a real map!"
+    "en": "Find yourself in a dispatch center and manage emergencies with your friends on a real map!",
+    "es": "¡Trabaja en una centralita y administra emergencias con tus amigos en un mapa real!",
+    "pl": "Wkrocz do centrum powiadamiania i razem z przyjaciółmi reaguj na zgłoszenia, korzystając z prawdziwej mapy!",
+    "it": "Trovati in un punto di risposta e gestisci le emergenze insieme ai tuoi amici su una mappa reale!",
+    "fr": "Répondez aux appels d’urgence et gérez les situations les plus délicates avec vos amis sur une carte en temps réel!",
+    "ru": "Принимайте вызовы и управляйте службами спасения вместе с друзьями на реальной карте!",
+    "da": "Du skal drive en alarmcentral og rykke ud til nødsituationer med dine venner – på et rigtigt kort!",
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/dist/metadata.json:46]:
```suggestion
    "da_DK": "Du skal drive en alarmcentral og rykke ud til nødsituationer med dine venner – på et rigtigt kort!",
```

File: websites/L/Leitstellenspiel/dist/metadata.json [hunk 4]
```diff
@@ -37,6 +37,18 @@
...
+    "no": "Sett deg på et svarpunkt og administrer nødstilfeller med vennene dine, på et ekte kart!",
+    "cs": "Staňte se dispečerem operačního střediska a vysílejte s přáteli záchranné jednotky k nehodám na mapě skutečného světa!",
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/dist/metadata.json:48]:
```suggestion
    "cs_CZ": "Staňte se dispečerem operačního střediska a vysílejte s přáteli záchranné jednotky k nehodám na mapě skutečného světa!",
```

File: websites/L/Leitstellenspiel/dist/metadata.json [hunk 5]
```diff
@@ -37,6 +37,18 @@
...
+    "tr": "Bir çağrı merkezinin başına geç ve acil durumları arkadaşlarınla gerçek bir harita üzerinde yönet!",
+    "pt": "Comande um Centro de Comando e comande ocorrências com os seus amigos num mapa real!",
+    "uk": "Приймайте виклики і управляйте службами порятунку разом із друзями на реальній карті!",
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/dist/metadata.json:51]:
```suggestion
    "uk_UA": "Приймайте виклики і управляйте службами порятунку разом із друзями на реальній карті!",
```

File: websites/L/Leitstellenspiel/dist/metadata.json [hunk 6]
```diff
@@ -37,6 +37,18 @@
...
+    "ja": "応答機関で指示を行い、自分の友達と共に実在するマップで緊急事態を解決しよう！"
```
Comment (Bas950) [inline websites/L/Leitstellenspiel/dist/metadata.json:52]:
```suggestion
    "ja_JP": "応答機関で指示を行い、自分の友達と共に実在するマップで緊急事態を解決しよう！"
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Slowlife01** [comment]
low quality

**Pull request closed without merging**